timrlink.net.CLI/DatabaseContext.cs

```diff
@@ -21,7 +28,14 @@ public DatabaseContext(string connectionString)
 
         protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
-            optionsBuilder.UseSqlServer(connectionString);
+            if (connectionString == null)
+            {
+                optionsBuilder.UseInMemoryDatabase(Guid.NewGuid().ToString());
```

Comment:
didn't notice the `Guid.NewGuid().ToString()` previously, so we always create a new Database? How do Migrations work in this case?